### ISSUE TITLE
specific QR code scan issue

### ISSUE THREAD

**Issue by Lenni112200** [corona-warn-app/cwa-documentation#880]
Issue: It‘s almost save to say that I have Covid. Do get my PCR certificate I’ve scanned the QR-code of the paper sheet that my doctor gave me. It worked. But after a few mins I deinstalled the Corona-Warn-App (don’t ask me why this was just stupid) and now as I reinstalled it, i can’t scan the QR code anymore. This does make sence of course because it says you can scan the QR code only once, but as I deinstalled the app, the scan result disappeared as well.
* Question: What can I do now to get my Covid certificate? I can’t scan the code anymore and the result of the first scan isn’t in my app anymore.

**Comment by Ein-Tim** [comment]
@Lenni112200 

To warn others via the app, you will have to call the verification hotline and request a TAN, which you can then enter in the app.
Please note that this is only possible for PCR tests.
Also note that, if you are using Android, uninstalling the app deletes all recorded contacts instantly IIRC, so even if you enter the TAN and warn others, nobody you met in the last says will get a warning.
Should you use iOS the situation is different, as contacts are not instantly deleted when deleting the app, so if you reinstalled the app directly after uninstalling it, it's quite certain that the contacts are still available. 

More information regarding how to request a TAN: https://www.coronawarn.app/en/faq/results/#international_phone_numbers

---

If you want to request an EU digital COVID certificate of recovery via the app, please note that this is **not** possible! You have to go to a Pharmacy or your doctor and show them the "Befundbrief" together with an ID. Then they can issue a certificate of recovery.

**Comment by Ein-Tim** [comment]
There was no feedback from @Lenni112200 in the last two weeks, I suggest to close this issue due to missing author feedback now.

**Comment by dsarkar** [comment]
@Lenni112200 Thanks for your question. We will close this issue now. Should have another question, or want to report a bug, or suggest an enhancement, please don't hesitate opening a new issue. Thanks again! Best wishes, DS

---
Corona-Warn-App Open Source Team
